libs/entry-components/configure-theming.md

```diff
@@ -0,0 +1,435 @@
+
+The new **custom-theme map**  should override the default theme that comes from a library. It contains a nested structure defining various aspects of a theme. Different key parameters can be passed in to extend it and customize it for our needs. The structure is organized into three main sections: **general**, **tables**, and **dialogs**. Each section contains specific configuration options.
+
+| Submap       | Description                                                             | Possible Options                      |
+|--------------|-------------------------------------------------------------------------|----------------------------------------|
+| **general**  | Influence the overall appearance and behavior of components. | `typography`, `fonts`, `spacing`, `inputs`, `buttons` |
+| **tables**   | Customize table component styling.                                       | `cells`, `rows`, `contents`            |
+| **dialogs**  | Set dialog font size.                                                   | `title`                                |
+
+```scss
+$custom-theme: (
+ general: (
+ // General configuration options go here
+ ),
+ tables: (
+ // Table-related configuration options go here
+ ),
+ dialogs: (
+ // Dialog-related configuration options go here
+ )
+);
+```
+
+### General Configuration
+
+The `general` section in the `$custom-theme` configuration contains settings that shape the overall look and behavior of the components.
+
+#### Density
+
+The density property typically accepts values like **-1**, **0** (default), and **1**, which correspond to different levels of density. For  [more info check](https://m2.material.io/design/layout/applying-density.html#usage):
+
+- `-1`: Reduces the spacing and makes elements more compact.
+- `0`: Default density according to the Material Design guidelines.
+- `1`: Increases spacing, providing a more relaxed and open layout.
+
+```scss
+$custom-theme: (
+ general: (
+ density: 0
+ )
+);
+```
+
+#### Colors
+
+Defines various color properties:
+
+- `primary`: Main color used in application, it sets the tone for the overall theme
+- `accent`: Highlight certain elements and create visual interest
+- `font`: Specifies the default font color.
+- `disabled`: Style elements that are in a disabled state:
+  - `foreground`: Text color for disabled elements
+  - `background`: Background color for disabled elements
+
+```scss
+$custom-theme: (
+ general: (
+  density: 0,
+  colors: (
+   primary: #2581C4,
+   accent: #EA518D,
+   font: #323232,
+   disabled: (
+    foreground: rgb(0 0 0 / .38),
+    background: rgb(0 0 0 / .12)
+   )
+  )
+ )
+);
+```
+
+#### Fonts
+
+Allow customization of typography and font styles based od Angular Material typography levels. Each font related property in configuration can have **family** and **size** values.
+
+- `hero-titles`: Define typography for h1, h2, h3, h4 elements.
+- `titles`: Typography h3, h4 elements
+- `body`: Define typography for base body text.
```

Comment:
Here we should explain that it makes no sense defining fonts without body font since by default it's gonna be applied everywhere (link to sentence where we mention that in docs).
It do makes sense if you skip defining some other font because body font will be picked up.
// Both titles font fallback to body font unless title's one's being set.

Reply:
I expanded this section, trying to better explain it